### PR TITLE
Log error once only in PerformanceAnalyzerTransportRequestHandler

### PR DESCRIPTION
**Pull request by yu-sun-77**
*Issue #, if available:*

*Description of changes:*
We're seeing same exception is being caught and logged for multiple times, decide to log it once to save log volume.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/transport/PerformanceAnalyzerTransportRequestHandler.java [hunk 1]
```diff
@@ -65,6 +65,7 @@ TransportChannel getChannel(T request, TransportChannel channel, Task task) {
     private TransportChannel getShardBulkChannel(T request, TransportChannel channel, Task task) {
         String className = request.getClass().getName();
         boolean bPrimary = false;
+        boolean logOnce = false;
```
Comment (yojs) [inline src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/transport/PerformanceAnalyzerTransportRequestHandler.java:68]: Let's make it a class member and not a method local.

**Review by yojs — CHANGES_REQUESTED** [review]

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/transport/PerformanceAnalyzerTransportRequestHandler.java (hunk 1 above)
Comment (yu-sun-77) [reply src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/transport/PerformanceAnalyzerTransportRequestHandler.java]: Done.

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/transport/PerformanceAnalyzerTransportRequestHandler.java [hunk 2]
```diff
@@ -66,6 +67,7 @@ private TransportChannel getShardBulkChannel(T request, TransportChannel channel
         String className = request.getClass().getName();
         boolean bPrimary = false;
 
+
```
Comment (sruti1312) [inline src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/transport/PerformanceAnalyzerTransportRequestHandler.java:70]: remove?

**Review by sruti1312 — APPROVED** [review]

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/transport/PerformanceAnalyzerTransportRequestHandler.java (hunk 2 above)
Comment (yu-sun-77) [reply src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/transport/PerformanceAnalyzerTransportRequestHandler.java]: Done

**Review by khushbr — APPROVED** [review]

**Review by sruti1312 — APPROVED** [review]

**Review by yojs — APPROVED** [review]

**Pull request merged**